.github/workflows/cli.yml

```diff
@@ -0,0 +1,116 @@
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'cpp/**'
+      - 'cli/**'
+      - '.github/workflows/ci.yml'
+      - '.github/workflows/cli.yml'
+concurrency:
+  group: ${{ github.repository }}-${{ github.event.number || github.head_ref || github.sha }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu:
+    name: Ubuntu 22.04 CLI
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.pull_request.title, 'WIP') && !github.event.pull_request.draft }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+          submodules: true
+
+    - name: Install dependencies
+      run: |
+
+        # install the latest arrow deb to test arrow
+        wget -c https://apache.jfrog.io/artifactory/arrow/"$(lsb_release --id --short | tr 'A-Z' 'a-z')"/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb \
+            -P /tmp/
+        sudo apt-get install -y /tmp/apache-arrow-apt-source-latest-"$(lsb_release --codename --short)".deb
+        sudo apt-get update -y
+        sudo apt install -y libarrow-dev \
+                            libarrow-dataset-dev \
+                            libarrow-acero-dev \
+                            libparquet-dev
+        sudo apt-get install -y ccache libcurl4-openssl-dev
+
+    - name: Install GraphAr CLI and Run Tests
+      working-directory: "cli"
+      run: |
+        pip install ./ -v
+        graphar --help
+        graphar check -p ../testing/neo4j/MovieGraph.graph.yml
+        graphar show -p ../testing/neo4j/MovieGraph.graph.yml -v Person
+        graphar show -p ../testing/neo4j/MovieGraph.graph.yml -es Person -e ACTED_IN -ed Movie
+        graphar import -c import.mini.yml
+# TODO: Add unit tests
+
+
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v4.0.1
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+
+  # macos:
+  #   name: ${{ matrix.architecture }} macOS ${{ matrix.macos-version }} CLI
+  #   runs-on: macos-${{ matrix.macos-version }}
+  #   if: ${{ !contains(github.event.pull_request.title, 'WIP') && github.event.pull_request.draft == false }}
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       include:
+  #         - architecture: AMD64
+  #           macos-version: "12"
+  #         - architecture: ARM64
+  #           macos-version: "14"
+  #   steps:
+  #   - uses: actions/checkout@v3
+  #     with:
+  #         submodules: true
+
+  #   - name: Install dependencies
+  #     run: |
+  #       brew bundle --file=cpp/Brewfile
+        
+    
+  #   - name: Build GraphAr And Run Tests
+  #     working-directory: "cli"
+  #     run: |
+  #       pip install ./
+  #       graphar --help
+  #       graphar check -p ../testing/neo4j/MovieGraph.graph.yml
+  #       graphar show -p ../testing/neo4j/MovieGraph.graph.yml -v Person
+  #       graphar show -p ../testing/neo4j/MovieGraph.graph.yml -es Person -e ACTED_IN -ed Movie
+  #       graphar import -c import.full.yml
```

Comment:
We can disable the job with `if: false`, which is considered better to me, rather than ignore them.

similar to example in [this doc](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow):

![image](https://github.com/user-attachments/assets/b36aa235-dbbe-4261-8106-41423bfb81db)

Reply:
Great suggestion!  Thanks  very much!